Core/NoOpRunner.Core/PlayerStates/PlayerState.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System;
+using System.Runtime.CompilerServices;
 
+[assembly: InternalsVisibleTo("NoOpRunner.Core.Tests")]
```

Comment:
Usually, this goes to the `AssemblyInfo.cs` file of the project

Reply:
Fixed